fairmq/nanomsg/FairMQTransportFactoryNN.cxx

```diff
@@ -63,7 +63,8 @@ FairMQPollerPtr FairMQTransportFactoryNN::CreatePoller(const FairMQSocket& cmdSo
 
 void FairMQTransportFactoryNN::Shutdown()
 {
-    nn_term();
+    // nn_term();
+    // see https://www.freelists.org/post/nanomsg/Getting-rid-of-nn-init-and-nn-term,8
 }
```

Comment:
Yes, it causes a nn test not to runnable twice. Once you call `nn_term()`, you cannot ever create a socket any more.